From c1c92a13f2d32c3e2c95cbb2cb1517d3b39e86ae Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibochen@cloudera.com>
Date: Thu, 29 Mar 2018 16:50:44 -0700
Subject: [PATCH 2787/2848] CLOUDERA-BUILD. CDH-66766: Disable YARN timeline
 and applicationhistoryservice unit tests in C5
 since they are not supported.

Change-Id: I8b9a4135d2bb4942dc15d41a092954c7b6fc8b0c
---
 cloudera/excludes.txt |   22 ++++++++++++++++++++++
 1 file changed, 22 insertions(+)

diff --git a/cloudera/excludes.txt b/cloudera/excludes.txt
index c38639f..b5fb84d 100644
--- a/cloudera/excludes.txt
+++ b/cloudera/excludes.txt
@@ -29,4 +29,26 @@ org/apache/hadoop/yarn/server/applicationhistoryservice/TestFileSystemApplicatio
 org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/TestContinuousScheduling.java
 org/apache/hadoop/yarn/applications/distributedshell/TestDistributedShell.java
 
+# Disable all timeline unit tests since we don't support it
+org/apache/hadoop/yarn/server/timeline/security/TestTimelineACLsManager.java
 org/apache/hadoop/yarn/server/timeline/security/TestTimelineAuthenticationFilter.java
+org/apache/hadoop/yarn/server/timeline/security/TestTimelineAuthenticationFilterInitializer.java
+org/apache/hadoop/yarn/server/timeline/webapp/TestCrossOriginFilter.java
+org/apache/hadoop/yarn/server/timeline/webapp/TestCrossOriginFilterInitializer.java
+org/apache/hadoop/yarn/server/timeline/webapp/TestTimelineWebServices.java
+org/apache/hadoop/yarn/server/timeline/webapp/TestTimelineWebServicesWithSSL.java
+org/apache/hadoop/yarn/server/timeline/TestGenericObjectMapper.java
+org/apache/hadoop/yarn/server/timeline/TestLeveldbTimelineStore.java
+org/apache/hadoop/yarn/server/timeline/TestMemoryTimelineStore.java
+org/apache/hadoop/yarn/server/timeline/TestTimelineDataManager.java
+
+# Disable all applicationhistoryservice unit tests because we don't support it
+org/apache/hadoop/yarn/server/applicationhistoryservice/webapp/TestAHSWebServices.java
+org/apache/hadoop/yarn/server/applicationhistoryservice/webapp/TestAHSWebApp.java
+org/apache/hadoop/yarn/server/applicationhistoryservice/TestApplicationHistoryClientService.java
+org/apache/hadoop/yarn/server/applicationhistoryservice/TestApplicationHistoryManagerImpl.java
+org/apache/hadoop/yarn/server/applicationhistoryservice/TestApplicationHistoryManagerOnTimelineStore.java
+org/apache/hadoop/yarn/server/applicationhistoryservice/TestApplicationHistoryServer.java
+org/apache/hadoop/yarn/server/applicationhistoryservice/TestFileSystemApplicationHistoryStore.java
+org/apache/hadoop/yarn/server/applicationhistoryservice/TestMemoryApplicationHistoryStore.java
+org/apache/hadoop/yarn/server/resourcemanager/ahs/TestRMApplicationHistoryWriter.java
-- 
1.7.9.5

